### DISCUSSION TITLE
11/18 (月) 20:00~ 全体ミーティング #5

### DISCUSSION THREAD

**Discussion by s1-haya** [42-pong/42-pong#73] [Meetings]
### 目次
- 進捗共有（FB / BE）
- 次回のタスク

**Comment by ayase-mstk** [comment]
## 次のフェーズ案
/games/

機能：
試合開始、得点表示、試合終了、試合一覧表示

通信：
試合終了（POST）、試合結果表示（GET）

役割：
frontend - ゲームロジック、画面表示
backend - 試合データ保存

API予想：
/api/games/
POST: point1, point2
GET : point1, point2, 終了時刻

**Comment by ayase-mstk** [comment]
次回タスク
- Backend
  - フェーズ案を基にDB設計、API設計、アーキテクチャあたりも
- Frontend
  - 引き続きベースづくり、各種調査、時間があればgame logicのところ
